### PR TITLE
Fix deploy cluster script

### PR DESCRIPTION
**Pull request by bowen5**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Fix sample aks-engine cluster definition and deployment script which used in e2e test.

**Which issue(s) this PR fixes**:
None.

**Special notes for your reviewer**:
None.

**Release note**:
```
None.
```

### CODE REVIEW

File: examples/az.json [hunk 1]
```diff
@@ -9,7 +9,6 @@
                 "useCloudControllerManager": true,
                 "customCcmImage": "gcrio.azureedge.net/google_containers/cloud-controller-manager-amd64:v1.15.0",
                 "customHyperkubeImage": "gcrio.azureedge.net/google_containers/hyperkube-amd64:v1.15.0",
-                "networkPolicy": "none",
```
Comment (feiskyer) [inline examples/az.json]: Could you explain why the "networkPolicy" filed is removed here?

Comment (bowen5) [reply examples/az.json]: I'm following https://github.com/bowen5/aks-engine/blob/master/docs/topics/clusterdefinitions.md and seems "none" is not a valid value for "networkPolicy" property for now. Or shall I change this value to "azure" or something else? Thank you.

**Comment by bowen5** [comment]
> Thanks for your pull request. Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> 📝 **Please follow instructions at https://git.k8s.io/community/CLA.md#the-contributor-license-agreement to sign the CLA.**
> 
> It may take a couple minutes for the CLA signature to be fully registered; after that, please reply here with a new comment and we'll verify. Thanks.
> 
> * If you've already signed a CLA, it's possible we don't have your GitHub username or you're using a different email address.  Check your existing CLA data and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> * If you signed the CLA as a corporation, please sign in with your organization's credentials at https://identity.linuxfoundation.org/projects/cncf to be authorized.
> * If you have done the above and are still having issues with the CLA being reported as unsigned, please log a ticket with the Linux Foundation Helpdesk: https://support.linuxfoundation.org/
> * Should you encounter any issues with the Linux Foundation Helpdesk, send a message to the backup e-mail support address at: [login-issues@jira.linuxfoundation.org](mailto:login-issues@jira.linuxfoundation.org)

Verify cncf-cla.

File: examples/az.json (hunk 1 above)
Comment (feiskyer) [reply examples/az.json]: OK, then let's remove it.

**Pull request merged**